### ISSUE TITLE
Patch Installed, Updates ran fine once. Now message is back.

### ISSUE THREAD

**Issue by HumbleSage0** [zeffy/wufuc#107]
Paste per your request:

<details>

## Operating System

```
BuildNumber=7601

Caption=Microsoft Windows 7 Ultimate 

MUILanguages={"en-US"}

OSArchitecture=64-bit

OSLanguage=1033

Version=6.1.7601

```

## Processor

```
Description=Intel64 Family 6 Model 158 Stepping 9

Family=198

Manufacturer=GenuineIntel

Name=Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz

NumberOfCores=4

NumberOfLogicalProcessors=8

ProcessorId=BFEBFBFF000906E9

Revision=

```

## Installed Hotfixes

```
HotFixID=KB2849697

InstalledOn=1/22/2017

HotFixID=KB2849696

InstalledOn=1/22/2017

HotFixID=KB2841134

InstalledOn=1/22/2017

HotFixID=KB2670838

InstalledOn=1/22/2017

HotFixID=KB971033

InstalledOn=1/22/2017

HotFixID=KB2506143

InstalledOn=3/3/2017

HotFixID=KB2479943

InstalledOn=1/22/2017

HotFixID=KB2491683

InstalledOn=1/22/2017

HotFixID=KB2506014

InstalledOn=1/22/2017

HotFixID=KB2506212

InstalledOn=1/22/2017

HotFixID=KB2509553

InstalledOn=1/22/2017

HotFixID=KB2532531

InstalledOn=1/22/2017

HotFixID=KB2533552

InstalledOn=1/22/2017

HotFixID=KB2533623

InstalledOn=1/22/2017

HotFixID=KB2534111

InstalledOn=1/22/2017

HotFixID=KB2552343

InstalledOn=1/22/2017

HotFixID=KB2560656

InstalledOn=1/22/2017

HotFixID=KB2564958

InstalledOn=1/22/2017

HotFixID=KB2579686

InstalledOn=1/22/2017

HotFixID=KB2585542

InstalledOn=1/22/2017

HotFixID=KB2604115

InstalledOn=1/22/2017

HotFixID=KB2620704

InstalledOn=1/22/2017

HotFixID=KB2621440

InstalledOn=1/22/2017

HotFixID=KB2631813

InstalledOn=1/22/2017

HotFixID=KB2653956

InstalledOn=1/22/2017

HotFixID=KB2654428

InstalledOn=1/22/2017

HotFixID=KB2656356

InstalledOn=1/22/2017

HotFixID=KB2667402

InstalledOn=1/22/2017

HotFixID=KB2676562

InstalledOn=1/22/2017

HotFixID=KB2685811

InstalledOn=1/22/2017

HotFixID=KB2685939

InstalledOn=1/22/2017

HotFixID=KB2690533

InstalledOn=1/22/2017

HotFixID=KB2698365

InstalledOn=1/22/2017

HotFixID=KB2705219

InstalledOn=1/22/2017

HotFixID=KB2706045

InstalledOn=1/22/2017

HotFixID=KB2727528

InstalledOn=1/22/2017

HotFixID=KB2729094

InstalledOn=1/22/2017

HotFixID=KB2729452

InstalledOn=1/22/2017

HotFixID=KB2731771

InstalledOn=1/22/2017

HotFixID=KB2736422

InstalledOn=1/22/2017

HotFixID=KB2742599

InstalledOn=1/22/2017

HotFixID=KB2758857

InstalledOn=1/22/2017

HotFixID=KB2770660

InstalledOn=1/22/2017

HotFixID=KB2786081

InstalledOn=1/22/2017

HotFixID=KB2789645

InstalledOn=1/22/2017

HotFixID=KB2798162

InstalledOn=1/22/2017

HotFixID=KB2807986

InstalledOn=1/22/2017

HotFixID=KB2813430

InstalledOn=1/22/2017

HotFixID=KB2834140

InstalledOn=1/22/2017

HotFixID=KB2836942

InstalledOn=1/22/2017

HotFixID=KB2836943

InstalledOn=1/22/2017

HotFixID=KB2840149

InstalledOn=1/22/2017

HotFixID=KB2840631

InstalledOn=1/22/2017

HotFixID=KB2847927

InstalledOn=1/22/2017

HotFixID=KB2861698

InstalledOn=1/22/2017

HotFixID=KB2862152

InstalledOn=1/22/2017

HotFixID=KB2862330

InstalledOn=1/22/2017

HotFixID=KB2862335

InstalledOn=1/22/2017

HotFixID=KB2864202

InstalledOn=1/22/2017

HotFixID=KB2868038

InstalledOn=1/22/2017

HotFixID=KB2868116

InstalledOn=1/22/2017

HotFixID=KB2871997

InstalledOn=1/22/2017

HotFixID=KB2882822

InstalledOn=1/22/2017

HotFixID=KB2884256

InstalledOn=1/22/2017

HotFixID=KB2888049

InstalledOn=1/22/2017

HotFixID=KB2892074

InstalledOn=1/22/2017

HotFixID=KB2893294

InstalledOn=1/22/2017

HotFixID=KB2894844

InstalledOn=1/22/2017

HotFixID=KB2900986

InstalledOn=1/22/2017

HotFixID=KB2911501

InstalledOn=1/22/2017

HotFixID=KB2912390

InstalledOn=1/22/2017

HotFixID=KB2929733

InstalledOn=1/22/2017

HotFixID=KB2931356

InstalledOn=1/22/2017

HotFixID=KB2937610

InstalledOn=1/22/2017

HotFixID=KB2943357

InstalledOn=1/22/2017

HotFixID=KB2957189

InstalledOn=1/22/2017

HotFixID=KB2965788

InstalledOn=1/22/2017

HotFixID=KB2968294

InstalledOn=1/22/2017

HotFixID=KB2972100

InstalledOn=1/22/2017

HotFixID=KB2972211

InstalledOn=1/22/2017

HotFixID=KB2973112

InstalledOn=1/22/2017

HotFixID=KB2973201

InstalledOn=1/22/2017

HotFixID=KB2973351

InstalledOn=1/22/2017

HotFixID=KB2977292

InstalledOn=1/22/2017

HotFixID=KB2978120

InstalledOn=1/22/2017

HotFixID=KB2978742

InstalledOn=1/22/2017

HotFixID=KB2984972

InstalledOn=1/22/2017

HotFixID=KB2991963

InstalledOn=1/22/2017

HotFixID=KB2992611

InstalledOn=1/22/2017

HotFixID=KB2999226

InstalledOn=1/22/2017

HotFixID=KB3003743

InstalledOn=1/22/2017

HotFixID=KB3004361

InstalledOn=1/22/2017

HotFixID=KB3004375

InstalledOn=1/22/2017

HotFixID=KB3010788

InstalledOn=1/22/2017

HotFixID=KB3011780

InstalledOn=1/22/2017

HotFixID=KB3019978

InstalledOn=1/22/2017

HotFixID=KB3020393

InstalledOn=1/22/2017

HotFixID=KB3021674

InstalledOn=1/22/2017

HotFixID=KB3022777

InstalledOn=1/22/2017

HotFixID=KB3023215

InstalledOn=1/22/2017

HotFixID=KB3030377

InstalledOn=1/22/2017

HotFixID=KB3033929

InstalledOn=1/22/2017

HotFixID=KB3035126

InstalledOn=1/22/2017

HotFixID=KB3035132

InstalledOn=1/22/2017

HotFixID=KB3037574

InstalledOn=1/22/2017

HotFixID=KB3042058

InstalledOn=1/22/2017

HotFixID=KB3042553

InstalledOn=1/22/2017

HotFixID=KB3045685

InstalledOn=1/22/2017

HotFixID=KB3046017

InstalledOn=1/22/2017

HotFixID=KB3046269

InstalledOn=1/22/2017

HotFixID=KB3055642

InstalledOn=1/22/2017

HotFixID=KB3059317

InstalledOn=1/22/2017

HotFixID=KB3060716

InstalledOn=1/22/2017

HotFixID=KB3067903

InstalledOn=1/22/2017

HotFixID=KB3067904

InstalledOn=1/22/2017

HotFixID=KB3071756

InstalledOn=1/22/2017

HotFixID=KB3072305

InstalledOn=1/22/2017

HotFixID=KB3072630

InstalledOn=1/22/2017

HotFixID=KB3074543

InstalledOn=1/22/2017

HotFixID=KB3075220

InstalledOn=1/22/2017

HotFixID=KB3076895

InstalledOn=1/22/2017

HotFixID=KB3078601

InstalledOn=1/22/2017

HotFixID=KB3080446

InstalledOn=1/22/2017

HotFixID=KB3084135

InstalledOn=1/22/2017

HotFixID=KB3086255

InstalledOn=1/22/2017

HotFixID=KB3092601

InstalledOn=1/22/2017

HotFixID=KB3093513

InstalledOn=1/22/2017

HotFixID=KB3097989

InstalledOn=1/22/2017

HotFixID=KB3101722

InstalledOn=1/22/2017

HotFixID=KB3108371

InstalledOn=1/22/2017

HotFixID=KB3108381

InstalledOn=1/22/2017

HotFixID=KB3108664

InstalledOn=1/22/2017

HotFixID=KB3108670

InstalledOn=1/22/2017

HotFixID=KB3109103

InstalledOn=1/22/2017

HotFixID=KB3109560

InstalledOn=1/22/2017

HotFixID=KB3110329

InstalledOn=1/22/2017

HotFixID=KB3115858

InstalledOn=1/22/2017

HotFixID=KB3122648

InstalledOn=1/22/2017

HotFixID=KB3123479

InstalledOn=1/22/2017

HotFixID=KB3124275

InstalledOn=1/22/2017

HotFixID=KB3126587

InstalledOn=1/22/2017

HotFixID=KB3127220

InstalledOn=1/22/2017

HotFixID=KB3135983

InstalledOn=1/22/2017

HotFixID=KB3138612

InstalledOn=1/22/2017

HotFixID=KB3138910

InstalledOn=1/22/2017

HotFixID=KB3139398

InstalledOn=1/22/2017

HotFixID=KB3139914

InstalledOn=1/22/2017

HotFixID=KB3142024

InstalledOn=1/22/2017

HotFixID=KB3146706

InstalledOn=1/22/2017

HotFixID=KB3146963

InstalledOn=1/22/2017

HotFixID=KB3149090

InstalledOn=1/22/2017

HotFixID=KB3150220

InstalledOn=1/22/2017

HotFixID=KB3155178

InstalledOn=1/22/2017

HotFixID=KB3156016

InstalledOn=1/22/2017

HotFixID=KB3156017

InstalledOn=1/22/2017

HotFixID=KB3156019

InstalledOn=1/22/2017

HotFixID=KB3159398

InstalledOn=1/22/2017

HotFixID=KB3161561

InstalledOn=1/22/2017

HotFixID=KB3161949

InstalledOn=1/22/2017

HotFixID=KB3161958

InstalledOn=1/22/2017

HotFixID=KB3163245

InstalledOn=1/22/2017

HotFixID=KB3170455

InstalledOn=1/22/2017

HotFixID=KB3177186

InstalledOn=1/22/2017

HotFixID=KB3177467

InstalledOn=1/22/2017

HotFixID=KB3210131

InstalledOn=1/22/2017

HotFixID=KB4014565

InstalledOn=4/12/2017

HotFixID=KB4019990

InstalledOn=9/29/2017

HotFixID=KB4040980

InstalledOn=9/29/2017

HotFixID=KB976902

InstalledOn=11/21/2010

HotFixID=KB4038777

InstalledOn=9/29/2017

```

## wuaueng.dll Properties

```
MD5 hash of file C:\Windows\System32\wuaueng.dll:
88 00 9d b9 e1 16 6b 6b 67 13 a8 58 c1 76 fe cd

SHA1 hash of file C:\Windows\System32\wuaueng.dll:
f2 4e 24 fe ff ef e8 40 b5 d7 ed 20 eb c9 9f b5 32 83 1c 30

SHA256 hash of file C:\Windows\System32\wuaueng.dll:
cb f4 c6 3d 3c 5d 14 af 3c 3f 0d 9c 48 e5 ac 9e 7a 43 23 bf b0 36 3e 99 48 fd 80 19 63 be 14 67

CreationDate=20170928225712.310400-240

FileSize=2651136

InstallDate=20170928225712.310400-240

LastAccessed=20170928230011.554400-240

LastModified=20170510111453.978724-240

Version=7.6.7601.23806

```
</details>

**Comment by zeffy** [comment]
Hey there, it looks like you're using an older version (<0.6) of wufuc, please download the [**latest version**](https://github.com/zeffy/wufuc/releases) (wufuc_setup_x64.msi) and try it out. It should solve any problems you are having. You may have to restart to fully remove the old patch after you install the new one.

For more information about how the new patch works, see the [README](https://github.com/zeffy/wufuc#how-it-works).

I'm going to close this issue for now, but if you have any more questions or issues please feel free to reopen it.

**Comment by HumbleSage0** [comment]
Zeffy,

 

Looks like that was it. I was installing from your .bat file after I followed a link from infopackets.com specifically:

 

https://www.infopackets.com/news/10108/how-fix-disable-unsupported-hardware-error-when-attempting-download-windows-updates

 

Maybe leave them a note stating that they aren’t pointing people to the right location. I also had to perform a hunt for the xdelta .exe which is not required under your new build. I’d hate to see people looking for this wonderful solution you’ve provided only to get frustrated and blame you for links they follow to old versions of stuff.

 

Keep up the excellent work!

- Rob
